MarsQA-1/Feature/Languages.feature

```diff
@@ -0,0 +1,71 @@
+Feature: My Profile Languages
+As a user I would be able to show what languages I know.
+So that the people seeking for languages can look at what details I hold.
+
+Background:
+	Given I login to the website
+	And All the exsisting records should be cleared
+
+Scenario: A_Add new language
```

Comment:
can you please make you use of tables 

For Example 

	Given I logged into the portal
	When I click on tab Education
	And I add my education 
	Then I am able to see my education'

Would  change to 


@Education
Scenario Outline: 1-Add education
	Given I logged into the portal
	When I click on tab Education
	And I add my education including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'
	Then I am able to see my education details including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'

	Examples:
	| Country     | University                      | Title  | Degree  | Graduation Year |
	| New Zealand | Whitireia Community Polytechnic | M.Tech | Diploma | 2021            |
	| Venezuela   | Hhutech University              | B.A    | Diploma | 2011            |

Reply:
Used tables every applicable places.